js/msp.js

```diff
@@ -63,7 +63,7 @@ var MSP_codes = {
     MSP_SET_SPECIAL_PARAMETERS: 99,
 
     // Multiwii MSP commands
-    MSP_IDENT:              100,
+    //MSP_IDENT:              100,
```

Comment:
I'd leave this in, just add a comment to it (like `MSP_BOX`), as this is still part of the 'official' MSP protocol spec.